src/components/Article.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticle } from "../utils/api";
+import "./Article.css"
+
+export const Article = () => {
+
+    const { article_id } = useParams();
+    const [currentArticle, setCurrArticle] = useState({})
+    const [loading, Isloading] = useState(true)
+
+    useEffect(() => {
+        getArticle(article_id)
+            .then((article) => {
+                setCurrArticle(article);
+                Isloading(false);
+            })
+    }, [])
+
+    if (loading) {
+        return <p>Loading...</p>
+    }
+
+    return (
+        <div className="article">
+            <div className="col"></div>
+            
+            <div className="Content">
+                <h2>{currentArticle.author}</h2>
+                <h1>{currentArticle.title}</h1>
+                <br />
+                <p>{currentArticle.body}</p>
+                <></>
+            </div>
+
+            <div className="col"></div>
+        </div>
+
+    )
```

Comment:
shouldn't be coding for the Article functionality (ticket 6) - please make a new branch and put that code in the new branch instead so this branch is specifically related to seeing articles by topic

Reply:
There is, but that branch was already in pull request because I accidentally did the tickets out of order ! Check this please
https://github.com/DanielTaylor1998/nc-news/pull/2